test/integration/good/code-gen/cpp.expected

```diff
@@ -4544,9 +4431,9 @@ binomialf(const T0__& phi_arg__, const T1__& theta_arg__,
     lpmf = Eigen::Matrix<local_scalar_t__, -1, 1>(1);
     stan::math::fill(lpmf, DUMMY_VAR__);
     
-    current_statement__ = 784;
+    current_statement__ = 773;
     assign(lpmf, 0.0, "assigning variable lpmf", index_uni(1));
-    current_statement__ = 785;
+    current_statement__ = 774;
```

Comment:
@seantalts I just want to safety check, I'm looking at the `locations_array__` and these seem to match up (773 pointing to an error on line 311 and 774 lining up with an error on line 312). Did we previously tick up the current statement for filling the object with NA values? If so then these current statement changes make sense, but wanted to double check